.github/workflows/themes-weekly-pull.yml

```diff
@@ -0,0 +1,46 @@
+name: themes-weekly-pull
+
+on:
+  schedule:
+    # run every friday at 5AM
+    - cron: '0 5 * * 5'
+
+jobs:
+  pull-from-jsonbin:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'

```

Comment:
In order to facilitate maintenance, I suggest defining this version via [an `env.NODE_VERSION` key](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env) at the top of the file, following the established convention for Arduino tooling workflows as was done here:

https://github.com/arduino/arduino-ide/blob/main/.github/workflows/compose-full-changelog.yaml#L11

Reply:
Thank you for the suggestion. I applied this convention here: fdf2e93a1de2b8e5b24df8ac36097ae57f6e254c.